CCDB/include/CCDB/CcdbApi.h

```diff
@@ -50,7 +50,18 @@ class CcdbApi //: public DatabaseInterface
    *
    * @param host The URL to the CCDB (e.g. "ccdb-test.cern.ch:8080")
    */
-  void init(std::string host);
+  void init(std::string const& host);
+
+  /**
+   * Initialize in local mode; Objects will be retrieved from snapshot
+   *
+   * @param snapshotpath (e.g. "/path/CCDBSnapshot/")
+   */
+  void initInSnapShotMode(std::string const& snapshotpath)
```

Comment:
Maybe this could be removed in favour of an extra parameter to a unified `init` method ? 
The current `init` takes the host URL as parameter. Maybe we could have a string with a prefix to specify the storage, i.e. "ccdb://" or "snapshot://", maybe rather "dir://" or "file://", à la  aliroot OCDB ?

Reply:
Implemented the "file://" approach.